react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -201,7 +201,22 @@ export default class JabraService extends VendorImplementation {
     }
   }
 
-  async answerCall (): Promise<void> {
+  async answerCall (conversationId: string, autoAnswer?: boolean): Promise<void> {
+    if (autoAnswer) {
+      this.pendingConversationId = conversationId;
+      try {
+        this.callLock = await this.callControl.takeCallLock();
+      } catch ({ message, type }) {
+        if (this.checkForCallLockError(message, type)) {
+          this.logger.info(message);
+          this.callControl.ring(false);
```

Comment:
do we need a return in here or can we just rely on the 
```
this.callControl.ring(false);
this.callControl.offHook(true);
```

below?

Reply:
Yanno, looking at this now and some other similar approaches, I think we should be fine of just relying on the one below.